components/sendgrid/sendgrid.app.js

```diff
@@ -1,6 +1,7 @@
 const axios = require("axios");
```

Comment:
Could this be updated to use Pipedream's axios? https://pipedream.com/docs/v1/pipedream-axios/#pipedream-platform-axios
```suggestion
const { axios } = require("@pipedream/platform");
```
or in ES6
```suggestion
import { axios } from "@pipedream/platform"
```